concordia/templates/admin/concordia/asset/change_form.html

```diff
@@ -33,7 +33,7 @@
 
     {% if transcriptions %}
         <table>
-            <caption>Transcription History</caption>
+            <caption>Transcription History - Current status: {{ original.transcription_status }}</caption>
```

Comment:
I'm wondering whether this should be in the history table rather than displayed above so it'd be visible even if there are no transcriptions. Perhaps it might also make sense to display them in reverse order on the theory that the most recent entry will be the most interesting.

Reply:
I moved the status out of the `if transcriptions` block so it'll always be there. I'm still curious why it doesn't show up in the default form since it's an attribute of the asset.